adam-core/src/test/scala/org/bdgenomics/adam/rdd/ADAMContextSuite.scala

```diff
@@ -224,6 +224,12 @@ class ADAMContextSuite extends ADAMFunSuite {
     assert(vcs.rdd.count === 7)
   }
 
+  sparkTest("read a vcf file generated by freebayes") {
```

Comment:
Small nit: can we change this from `generated by freebayes` to `with Number=A format fields`? I'd like the test to be more clearly linked to the bug rather than the data source.